content/articles/understanding-facial-recognition-using-local-binary-pattern-histogram-(lbph)-algorithm/index.md

```diff
@@ -0,0 +1,92 @@
+The face recognition algorithm generally operates in one of two modes: 
+
+1.	*Authentication of a facial image:* This mode does facial recognition by a 1x1 comparison. The comparison is done between an input image and a specific image within the database. In many cases, this is the face that requires authentication at the time of this mode of facial recognition.
+
+2.	*Face recognition:* in this mode, it is a 1xN, a comparison of the input face image with all the pictures that have been saved in the database to output the images of the user which conforms to the input face image.
+
+### The stages of face recognition using LBPH 
+For face recognition, the LBPH algorithm follows some steps. These steps can be carried out in two stages, as follows: 
+#### The learning of the algorithm
+The first step is the learning of the algorithm. It makes use of a data-set of images of the people to be included in the recognition process. Each image is given a unique ID as either a number or the name of a person so that the algorithm can use this information to identify the image and export the output. The pictures of the same person are always placed under the same ID. 
+#### Computational steps
+1.	**The application of the LBP operation:** is the first step of the computational steps. Here, an intermediate image has been created to better represent the original image through a sliding window concept, taking into account two parameters: the `neighbor` and the `radius`. New values are created in the form of binary by comparing the 8 `neighbor` values to the threshold value. For each `neighbor` value greater than the threshold value, the value is set to 1 and 0 for every `neighbor` value less than the threshold value. This forms a matrix of binary numbers excluding the threshold. A central value of the matrix is created by the conversion of the binary number to a decimal value which corresponds to the pixels of the original image. For a better representation of the characteristics of the original image.
+
+2.	**To Extract Histograms:** The image obtained in step is divided into multiple grids, with the help of the Grid parameters `X` and `Y`. This image is in grayscale, each of the histograms of each of the grids is to represent the intensity of the occurrences of each pixel. Each histogram is then combined to create a new histogram that represents the attributes of the original image.
+
+3.	**Accurate face recognition:** Each one made a histogram for an image in the training data set. Two histograms are compared to output the image with the closest histogram matches to an input image. This output is the ID or name of the image. This algorithm also returns a **`confidence`**' measurement which is the calculated distance. The correctness of the algorithm in recognizing the image is estimated automatically by the `confidence` and the threshold. The correctness is indicated by a confidence value that is lower than the defined threshold. 
```

Comment:
Generally, since a higher value of confidence is typically associated with greater certainty, one might ask, why is a lower value of confidence preferred in the context of LBPH face recognition?